core/src/main/kotlin/pages/ContentNodes.kt

```diff
@@ -383,7 +383,8 @@ enum class TokenStyle : Style {
 
 enum class TextStyle : Style {
     Bold, Italic, Strong, Strikethrough, Paragraph,
-    Block, Span, Monospace, Indented, Cover, UnderCoverText, BreakableAfter, Breakable, InlineComment, Quotation
+    Block, Span, Monospace, Indented, Cover, UnderCoverText, BreakableAfter, Breakable, InlineComment, Quotation,
+    RightAligned
```

Comment:
I think it's better to put the common word first, i.e `AlignedRight`, so that when you type `Aligned`, you'll see more options in autocomplete, like `AlignedLeft`, `AlignedCenter`, etc